src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -97,33 +98,33 @@ UsgsAstroFrameSensorModel::UsgsAstroFrameSensorModel() {
   m_startingDetectorSample = 0.0;
   m_startingDetectorLine = 0.0;
   m_targetName = "";
-  m_ifov = 0.0;
+  m_ifov = 6;
   m_instrumentID = "";
   m_focalLengthEpsilon = 0.0;
 
-  m_ccdCenter[0] = 0.0;
-  m_ccdCenter[1] = 0.0;
+  m_ccdCenter[0] = 7.5;
+  m_ccdCenter[1] = 7.5;
 
   m_line_pp = 0.0;
   m_sample_pp = 0.0;
 
   m_odtX.assign(10, 0.0);
   m_odtY.assign(10, 0.0);
 
-  m_originalHalfLines = 0.0;
+  m_originalHalfLines = 8.0;
```

Comment:
should be 0.0 - > for all of these changes, we want to keep the defaults as 0.0

Reply:
These changes were only because image2ground relies on params not in the ISD anymore. I imagine the thing to do is reset them and let the tests fail and in a separate PR, someone else can make those changes.